src/main/kotlin/camp/nextstep/edu/racingcar/racing/Car.kt

```diff
@@ -34,7 +38,13 @@ class Car(val id: Int, private val engine: Engine = RandomlyEngine()) {
         private val random = Random(Random.Default.nextInt())
 
         override fun speed(): Int {
-            return if (random.nextInt(0, 10) >= 4) 1 else 0
+            return if (random.nextInt(POWER_MIN, POWER_MAX) >= MOVABLE_POWER_MIN) 1 else 0
```

Comment:
Nit: 요런 표헌도 가능합니다. 
```suggestion
            return if ((POWER_MIN .. POWER_MAX).random() >= MOVABLE_POWER_MIN) 1 else 0
```
그런데 전 `random` 프러퍼티가 `RandomlyEngine`  객체의 중요한 속성으로 보여서 지금 구현도 맘에 들어요.